tfhe/src/integer/gpu/server_key/radix/mod.rs

```diff
@@ -293,6 +294,86 @@ impl CudaServerKey {
         carry_out
     }
 
+    /// # Safety
+    ///
+    /// - `streams` __must__ be synchronized to guarantee computation has finished, and inputs must
+    ///   not be dropped until streams is synchronized
+    pub(crate) unsafe fn sub_and_propagate_single_carry_assign_async<T>(
```

Comment:
This function should be located in sub.rs instead of here. Is add_and_propagate_single_carry exposed here? This would be a mistake it would be better to have it in add.rs as well actually. :slightly_smiling_face: